Date: Sun, 30 Nov 2025 17:05:12 -0100
Description: render title color the same as inactive color
--- bar.cpp.orig	2025-11-30 17:04:17.841334849 +0100
+++ src/bar.cpp	2025-11-30 17:01:21.227109605 +0100
@@ -225,7 +225,7 @@ void Bar::render()
 	_x = 0;
 
 	renderTags();
-	setColorScheme(_selected ? colorActive : colorInactive);
+	setColorScheme(_selected ? colorInactive : colorInactive);
 	renderComponent(_layoutCmp);
 	renderStatus();
 
